site/content/docs/5.2/examples/form/form.css

```diff
@@ -0,0 +1,3 @@
+.footer > * .navbar-nav .nav-link.active {
```

Comment:
IMO this modification should be directly added to our footers shouldn't it?

Reply:
Technically yes. But I decided to keep it in a distinct file for now because there is still no consensus about it.

Reply:
OK so if there's consensus about it, you can drop it from this PR to respect consistency. And create an issue to track it.